java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -3498,6 +3498,10 @@ public final Scalar getScalarElement(int index) {
     return new Scalar(getType(), getElement(getNativeView(), index));
   }
 
+  public final ColumnVector applyBooleanMask(ColumnView booleanMaskView) {
+    return new ColumnVector(applyBooleanMask(getNativeView(), booleanMaskView.getNativeView()));
```

Comment:
Here can add an assertion for the types of the input and this before calling into the native.

Reply:
Better to add the description for this public API, you may simply copy it from the native API's doc.